frontend/src/components/common/SideBar.tsx

```diff
@@ -149,10 +213,35 @@ export default function SideBar() {
                   {item.label}
                 </Button>
               ))}
+
+              <Spacer y={72} />
+              <Spacer y={6} />
+              { isLoggedIn ?
```

Comment:
I have created a log out button in the UserDropdown component, so we should discard this log out button here and the Spacer above it. Since this is causing a bad UI rendering on my computer (and not just my computer), see example screenshot:
![image](https://github.com/tryyang2001/CS3213-Frontend-Management-System/assets/69421979/8a7227d5-54a3-48e8-822d-a80d004f6c2b)
Also there is no need to render a button for sign in, since if the user has no token/user context, it will redirect to login page.